src/app/search/search-results/search-results.component.scss

```diff
@@ -14,6 +14,19 @@
  *    limitations under the License.
  */
 
+mat-tab-group.homeComponent {
+  border: 1px solid #cfd1d7;
+  border-bottom-right-radius: 15px;
```

Comment:
Avoid pixels in css. See [10th item or so](https://github.com/dockstore/dockstore/wiki/Dockstore-Frontend-Opinionated-Style-Guide).

Reply:
Done.